apps/src/javalab/JavalabView.jsx

```diff
@@ -109,11 +112,16 @@ class JavalabView extends React.Component {
   };
 
   isLeftSideVisible = () => {
+    const {
+      shortInstructions,
+      longInstructions,
+      hasContainedLevels
+    } = this.props;
     // It's possible that a console level without instructions won't have
     // anything to show on the left side.
     return (
       this.props.viewMode !== CsaViewMode.CONSOLE ||
-      !!this.props.longInstructions
+      hasInstructions(shortInstructions, longInstructions, hasContainedLevels)
```

Comment:
Javalab levels never have short instructions, so we can probably just pass null instead of `shortInstructions` here

Reply:
Can do -- for my understanding, why is that?

Reply:
they are only used for K-5 I think